src/EnergyPlus/HVACManager.cc

```diff
@@ -2428,7 +2452,7 @@ void ReportAirHeatBalance(EnergyPlusData &state)
     }
 
     // Reports zone exhaust loss by exhaust fans
-    for (ZoneLoop = 1; ZoneLoop <= state.dataGlobal->NumOfZones; ++ZoneLoop) { // Start of zone loads report variable update loop ...
+    for (int ZoneLoop = 1; ZoneLoop <= state.dataGlobal->NumOfZones; ++ZoneLoop) { // Start of zone loads report variable update loop ...
```

Comment:
Scoped variables; it's  little things like this that can bring a smile as I review.